taprpc/mintrpc/mint.proto

```diff
@@ -16,6 +16,20 @@ service Mint {
     */
     rpc MintAsset (MintAssetRequest) returns (MintAssetResponse);
 
+    /* tapcli `assets mint fund`
```

Comment:
Reminder to implement those two new commands in the CLI as well in this PR, otherwise this breaks the API docs generator.